core/modules/cart/helpers/productChecksum.ts

```diff
@@ -8,8 +8,9 @@ const replaceNumberToString = obj => {
   Object.keys(obj).forEach(key => {
     if (typeof obj[key] === 'object') {
```

Comment:
This will also be true for null (and fail as Object.keys(obj.foreach will fail then).

as typeof null === "object"

Reply:
you are right, thanks!